src/AccessibilityInsights/Modes/DialogContainerModeControl.xaml.cs

```diff
@@ -29,7 +29,7 @@ protected override AutomationPeer OnCreateAutomationPeer()
             return new CustomControlOverridingAutomationPeer(this, Properties.Resources.LocalizedControlType_Page);
         }
 
-        private void HideControl(ContainedDialog dialog) => Dispatcher.Invoke(() =>
+        public void HideControl(ContainedDialog dialog) => Dispatcher.Invoke(() =>
```

Comment:
intentional to provide a imperative show/hide experience rather than a blocking showdialog where the dialog is in charge of executing the operation